src/library/ksTable/ksActionsManager.jsx

```diff
@@ -141,21 +173,36 @@ const KsActionsManager = ({ entityName, entityStructure = {}, actions = mockActi
   const handleAdd = form => {
     entityActions[entityAction](form);
     // TODO the close of the modal should wait all the flow or have a notification for handle errors
-    setIsOpen(false);
+    setModals([false, deleteModal]);
+  };
+
+  const handleDelete = id => {
+    entityActions[entityAction]({ id });
+    // TODO the close of the modal should wait all the flow or have a notification for handle errors
```

Comment:
You should address to the standard `// TODO:` to be read by the IDE or different tools

```suggestion
    // TODO: the close of the modal should wait for all the flow or have a notification for handle errors
```